cache-config/testing/ort-tests/config/config.go

```diff
@@ -32,95 +32,143 @@ import (
+	// no matter what "Priv Level" and/or Permissions you give the user!
 	Extension string `json:"extension" envconfig:"TO_USER_EXTENSION"`
 }
 
-// TrafficOpsDB - config section
+// TrafficOpsDB is the section of a Config dealing with Traffic Ops DB-related
+// information.
 type TrafficOpsDB struct {
-	// Name - Traffic Ops Database name where the test data will be setup
+	// Name is the name of a PostgreSQL database used by the testing Traffic Ops
+	// instance.
 	Name string `json:"dbname" envconfig:"TODB_NAME"`
 
-	// Hostname - Traffic Ops Database hostname where Postgres is running
+	// Hostname is the network hostname where the Traffic Ops database is
+	// running.
 	Hostname string `json:"hostname" envconfig:"TODB_HOSTNAME"`
 
-	// User - database user that Traffic Ops is using to point to the 'to_test' database
+	// User is the name of a PostgreSQL user/role that has permissions to
+	// manipulate the database identified in Name.
 	User string `json:"user" envconfig:"TODB_USER"`
 
-	// Password - database password for the User above
+	// Password is the password for the PostgreSQL user/role given in User.
 	Password string `json:"password" envconfig:"TODB_PASSWORD"`
 
-	// Port - Postgres port running that has the to_test schema
+	// Port is the port on which PostgreSQL listens for connections to the
+	// database used by the testing Traffic Ops instance.
 	Port string `json:"port" envconfig:"TODB_PORT"`
 
-	// DBType - will be 'Pg' by default but tells the Golang database driver
-	//          the database type
+	// DBType is a Go database/sql driver name to use when connecting to the
+	// Traffic Ops testing instance's database. This MUST be "Pg" (or
+	// omitted/blank/null, which uses the default value of "Pg").
+	//
+	// Deprecated: Since Traffic Ops 3.0, only PostgreSQL databases are
+	// supported, so this field has no purpose and will probably be removed at
+	// some point.
 	DBType string `json:"type" envconfig:"TODB_TYPE"`
 
-	// SSL - Flag that tells the database driver that the Postgres instances has TLS enabled
+	// SSL instructs the database driver that the PostgreSQL instance used by
+	// the tests requires SSL-secured connections.
 	SSL bool `json:"ssl" envconfig:"TODB_SSL"`
 
-	// Description - database description
+	// Description is a textual description of the database.
+	//
+	// Deprecated: This unused field serves no purpose, and will likely be
+	// removed in the future.
 	Description string `json:"description" envconfig:"TODB_DESCRIPTION"`
 }
 
-// Default - config section
+// Default represents the default values of a set of options that can be
+// overridden by command-line options.
 type Default struct {
-	Session            Session   `json:"session"`
-	Log                Locations `json:"logLocations"`
-	IncludeSystemTests bool      `json:"includeSystemTests"`
+	Session Session   `json:"session"`
+	Log     Locations `json:"logLocations"`
+	// Deprecated: This field has no effect or known purpose.
```

Comment:
Did you mean to make this not follow to GoDoc? (A complete sentence starting with the symbol name.)

Reply:
Sort of, I wanted to skip the description since it has no known purpose. I'll add it though.